scripts/run-live-build.sh

```diff
@@ -89,11 +89,8 @@ cp -r "$TOP/live-build/auto" "$build_dir"
 rsync -a --exclude="hooks" "$TOP/live-build/config" "$build_dir"
 mkdir -p "$build_dir/config/hooks"
 cp -r "$TOP/live-build/config/hooks/configuration/." "$build_dir/config/hooks"
-if [[ "$RUN_TYPE" == "$ALL_RUN_TYPE" ]]; then
-	cp -r "$TOP/live-build/config/hooks/$UPGRADE_RUN_TYPE/." "$build_dir/config/hooks"
+if [[ "$RUN_TYPE" == "$VM_RUN_TYPE" ]]; then
```

Comment:
This should be done for both `VM_RUN_TYPE` and `ALL_RUN_TYPE` (hence the failure in your test)